sierra_adapter/report_adapter_progress.py

```diff
@@ -206,9 +209,32 @@ def chunks(iterable, chunk_size):
 
         print('')
 
-    print('')
-    print('-' * 79)
-    print('')
-    print('If there are gaps in the report, you can build new windows for the readers:')
-    print('')
-    print(f'$ python {sys.argv[0].replace("report_adapter_progress.py", "build_missing_windows.py")}')
+        # Now we've consolidated the markers in S3, produce a second copy
+        # of the report that "summarises" the output.
+        report = build_report(bucket=BUCKET, resource_type=resource_type)
+        final_reports[resource_type] = [iv for (iv, _) in report]
+
+    # We only need to give instructions for building missing windows if
+    # there are gaps in the run.
+    #
+    if len(final_reports['bibs']) > 1 or len(final_reports['items']) > 1:
+        print('')
+        print('-' * 79)
+        print('')
+        print('You can build new windows for the gaps:')
+        print('')
+        print(f'$ python {sys.argv[0].replace("report_adapter_progress.py", "build_missing_windows.py")}')\
+
+    # If the run completed successfully, we can purge the associated DLQs.
+    #
```

Comment:
I'm sure purging the DLQs is useful, I wonder though about it being done by default in a script called report_adapter_progress.

Reply:
Good point. I could hide it behind a `--purge-dlqs` flag:

* If you don’t pass the flag, it says “There are X messages on the DLQs”.
* If you do pass the flags, it empties the DLQs.

Reply:
nice, yes thanks